service/src/main/java/uk/gov/hmcts/reform/fpl/service/EventService.java

```diff
@@ -1,17 +1,20 @@
 package uk.gov.hmcts.reform.fpl.service;
 
 import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.ApplicationEventPublisher;
 import org.springframework.stereotype.Service;
 
 @Service
+@Slf4j
 @RequiredArgsConstructor(onConstructor = @__(@Autowired))
 public class EventService {
 
     private final ApplicationEventPublisher applicationEventPublisher;
 
     public void publishEvent(Object event) {
+        log.info("Publishing event {}", event.getClass().getSimpleName());
```

Comment:
something left over?

Reply:
Yeah, was checking that events were still firing. Do you want me to get rid of it or do you think it is a worthwhile addition?